From afc7e69688808d38d304df380081a5615d528ceb Mon Sep 17 00:00:00 2001
From: Joshua Wong Weng Onn <joshua.weng.onn.wong@intel.com>
Date: Mon, 4 Jun 2018 13:27:14 +0800
Subject: [PATCH 31/31] ASoC: Intel: Change be_id variable in machine drivers

The be_id variable has been replaced with id. Hence, change be_id to id
for the following machine drivers.

1. lhcrb_dummy_i2s
2. lhcrb_wm8731_i2s
3. lhcrb_aic3107M_i2s
4. lhcrb_aic3107S_i2s
5. apli_lhcrb_generic_i2s

Signed-off-by: Joshua Wong Weng Onn <joshua.weng.onn.wong@intel.com>
---
 sound/soc/intel/boards/apli_lhcrb_aic3107M.c |  6 +++---
 sound/soc/intel/boards/apli_lhcrb_aic3107S.c |  6 +++---
 sound/soc/intel/boards/apli_lhcrb_dummy.c    |  4 ++--
 sound/soc/intel/boards/apli_lhcrb_generic.c  | 16 ++++++++--------
 sound/soc/intel/boards/apli_lhcrb_wm8731.c   |  4 ++--
 5 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/sound/soc/intel/boards/apli_lhcrb_aic3107M.c b/sound/soc/intel/boards/apli_lhcrb_aic3107M.c
index ac80546..e0448f4 100644
--- a/sound/soc/intel/boards/apli_lhcrb_aic3107M.c
+++ b/sound/soc/intel/boards/apli_lhcrb_aic3107M.c
@@ -281,7 +281,7 @@ static int aic3107_be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 	{
 		/* SSP0 - Codec */
 		.name = "SSP0-Codec",
-		.be_id = 0,
+		.id = 0,
 		.cpu_dai_name = "SSP0 Pin",
 		.codec_name = "tlv320aic3x-codec.3-0018",
 		.codec_dai_name = "tlv320aic3x-hifi",
@@ -301,7 +301,7 @@ static int aic3107_be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 	{
 		/* SSP1 - Codec */
 		.name = "SSP1-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -315,7 +315,7 @@ static int aic3107_be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 	{
 		/* SSP3 - Codec */
 		.name = "SSP3-Codec",
-		.be_id = 2,
+		.id = 2,
 		.cpu_dai_name = "SSP3 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
diff --git a/sound/soc/intel/boards/apli_lhcrb_aic3107S.c b/sound/soc/intel/boards/apli_lhcrb_aic3107S.c
index 952eb01..ef79ca7 100644
--- a/sound/soc/intel/boards/apli_lhcrb_aic3107S.c
+++ b/sound/soc/intel/boards/apli_lhcrb_aic3107S.c
@@ -281,7 +281,7 @@ static int aic3107_be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 	{
 		/* SSP0 - Codec */
 		.name = "SSP0-Codec",
-		.be_id = 0,
+		.id = 0,
 		.cpu_dai_name = "SSP0 Pin",
 		.codec_name = "tlv320aic3x-codec.3-0018",
 		.codec_dai_name = "tlv320aic3x-hifi",
@@ -301,7 +301,7 @@ static int aic3107_be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 	{
 		/* SSP1 - Codec */
 		.name = "SSP1-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -315,7 +315,7 @@ static int aic3107_be_hw_params_fixup(struct snd_soc_pcm_runtime *rtd,
 	{
 		/* SSP3 - Codec */
 		.name = "SSP3-Codec",
-		.be_id = 2,
+		.id = 2,
 		.cpu_dai_name = "SSP3 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
diff --git a/sound/soc/intel/boards/apli_lhcrb_dummy.c b/sound/soc/intel/boards/apli_lhcrb_dummy.c
index a9accc8..78fea08 100644
--- a/sound/soc/intel/boards/apli_lhcrb_dummy.c
+++ b/sound/soc/intel/boards/apli_lhcrb_dummy.c
@@ -159,7 +159,7 @@ static int apli_lhcrb_dummy_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP1 - Codec */
 		.name = "SSP1-Codec",
-		.be_id = 0,
+		.id = 0,
 		.cpu_dai_name = "SSP1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -176,7 +176,7 @@ static int apli_lhcrb_dummy_startup(struct snd_pcm_substream *substream)
 	{
 		/* SSP3 - Codec */
 		.name = "SSP3-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP3 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
diff --git a/sound/soc/intel/boards/apli_lhcrb_generic.c b/sound/soc/intel/boards/apli_lhcrb_generic.c
index a6a3fb5..eae5f21 100644
--- a/sound/soc/intel/boards/apli_lhcrb_generic.c
+++ b/sound/soc/intel/boards/apli_lhcrb_generic.c
@@ -118,7 +118,7 @@ static int apli_ssp_northwest_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	char *gpio_addr = NULL;
 	u32 gpio_value = NORTHWEST_GPIO_VALUE;
 
-	switch (rtd->dai_link->be_id) {
+	switch (rtd->dai_link->id) {
 	case 0:
 		gpio_addr = (void *)ioremap_nocache(NORTHWEST_BASE
 				+ I2S_1, 0x20);
@@ -157,7 +157,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	char *gpio_addr = NULL;
 	u32 gpio_value = WEST_GPIO_VALUE;
 
-	switch (rtd->dai_link->be_id) {
+	switch (rtd->dai_link->id) {
 	case 4:
 		gpio_addr = (void *)ioremap_nocache(WEST_BASE + I2S_5, 0x20);
 		break;
@@ -433,7 +433,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	{
 		/* SSP1 - Codec */
 		.name = "SSP0-Codec",
-		.be_id = 0,
+		.id = 0,
 		.cpu_dai_name = "SSP0 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -450,7 +450,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	{
 		/* SSP2 - Codec */
 		.name = "SSP1-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -467,7 +467,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	{
 		/* SSP3 - Codec */
 		.name = "SSP2-Codec",
-		.be_id = 2,
+		.id = 2,
 		.cpu_dai_name = "SSP2 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -484,7 +484,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	{
 		/* SSP4 - Codec */
 		.name = "SSP3-Codec",
-		.be_id = 3,
+		.id = 3,
 		.cpu_dai_name = "SSP3 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -501,7 +501,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	{
 		/* SSP5 - Codec */
 		.name = "SSP4-Codec",
-		.be_id = 4,
+		.id = 4,
 		.cpu_dai_name = "SSP4 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -518,7 +518,7 @@ static int apli_ssp_west_gpio_init(struct snd_soc_pcm_runtime *rtd)
 	{
 		/* SSP6 - Codec */
 		.name = "SSP5-Codec",
-		.be_id = 5,
+		.id = 5,
 		.cpu_dai_name = "SSP5 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
diff --git a/sound/soc/intel/boards/apli_lhcrb_wm8731.c b/sound/soc/intel/boards/apli_lhcrb_wm8731.c
index d3c915c..c98aed0 100644
--- a/sound/soc/intel/boards/apli_lhcrb_wm8731.c
+++ b/sound/soc/intel/boards/apli_lhcrb_wm8731.c
@@ -193,7 +193,7 @@ static int apli_wm8731_hw_params(struct snd_pcm_substream *substream,
 	{
 		/* SSP1 - Codec */
 		.name = "SSP1-Codec",
-		.be_id = 0,
+		.id = 0,
 		.cpu_dai_name = "SSP1 Pin",
 		.codec_name = "snd-soc-dummy",
 		.codec_dai_name = "snd-soc-dummy-dai",
@@ -207,7 +207,7 @@ static int apli_wm8731_hw_params(struct snd_pcm_substream *substream,
 	{
 		/* SSP3 - Codec */
 		.name = "SSP3-Codec",
-		.be_id = 1,
+		.id = 1,
 		.cpu_dai_name = "SSP3 Pin",
 		.codec_name = "wm8731.3-001a",
 		.codec_dai_name = "wm8731-hifi",
-- 
1.9.1

